tests/c_cpp/src/main.c

```diff
@@ -0,0 +1,7 @@
+int main()
+{
+    int array[5];
```

Comment:
Codacy found an issue: [Variable 'array' is not assigned a value.](https://app.codacy.com/gh/cnescatlab/sonar-scanner/pullRequest?prid=6081579)